src/icalendar/alarms.py

```diff
@@ -200,6 +199,8 @@ def add_component(self, component: Alarm | Parent):
         Events and Todos are added as a parent and all
         their alarms are added, too.
         """
+        from icalendar.cal.event import Event
+        from icalendar.cal.todo import Todo
```

Comment:
It's possible I just don't understand the nuance of the changes in this file. I lack familiarity and it's not obvious to me.